src/form-question-types.rst

```diff
@@ -2399,4 +2400,44 @@ storing the values for later use.
 
 For more details, see :ref:`calculations`.
 
---------
\ No newline at end of file
+.. _background-audio:
+
+Background audio
+~~~~~~~~~~~~~~~~~
+
+type
+  :tc:`background-audio`
+
+.. versionadded:: 1.30
+
+  `ODK Collect v1.30.0 <https://github.com/getodk/collect/releases/tag/v1.30.0>`_, `pyxform` v1.4.0, `XLSForm Online` v2.4.0
+
+.. seealso::
+  :doc:`Logging enumerator behavior <form-audit-log>`, :ref:`audio questions <audio>`
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name
+
+  background-audio, my_recording
+
+When a form includes a question of type :tc:`backgroud-audio`, audio is recorded while the form is open and attached to the form submission as a single audio file. These recordings can be used for quality assurance, training, transcription, and more. Use background recording instead of an :ref:`audio question <customizing-audio-quality>` when you want to make sure to record everything that happens during form filling.
+
+Before adding background audio recording to your form, make sure that you have a plan for following local laws around audio recording. We generally recommend including a note at the beginning of the form to remind data collectors and participants that they are being recorded and to describe the purpose of the recording. Depending on the context, you may be required to ask for the consent of every person in speaking range of the microphone. <Examples>
```

Comment:
This and the next paragraph feel like they might suit being a "tip" right? I feel like this whole section is quite long, and it'd be good to break up the opinionated/advisory stuff from the core info.